src/Application.vala

```diff
@@ -31,10 +31,25 @@ public class Marlin.Application : Gtk.Application {
     private const uint MAX_WINDOWS = 25;
 
     public int window_count { get; private set; }
+    public Settings marlin_app_settings { get; construct; }
+    public Settings marlin_icon_view_settings { get; construct; }
+    public Settings marlin_list_view_settings { get; construct; }
+    public Settings marlin_column_view_settings { get; construct; }
+    public Settings gnome_interface_settings { get; construct; }
+    public Settings gnome_privacy_settings { get; construct; }
+    public Settings gtk_file_chooser_settings { get; construct; }
```

Comment:
Should these be constructed statically instead? Then you wouldn't have to reference the instance as in `slot.window.marlin_app.marlin_app_settings` which is kind of verbose

Reply:
Done.